include/NovelRT/Atom.h

```diff
@@ -93,6 +93,8 @@ namespace NovelRT
 
         static Atom GetNextEcsGraphicsPipelineId() noexcept;
 
+        static Atom GetNextEcsInputActionId() noexcept;
```

Comment:
Is this still used? Looks like not from my search. We could probably delete it.

Reply:
Removeddddd